### DISCUSSION TITLE
Bevy engine exports web files. How to close the (print)console or the (print)console does not output the user's directory?

### DISCUSSION THREAD

**Discussion by ysypnbh** [bevyengine/bevy#16305] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/12f919a7-89e3-4f69-8fdd-16cfe24463d1)

**Comment by rparrett** [comment, marked as answer]
Couple options:
- [--remap-path-prefix](https://users.rust-lang.org/t/how-to-properly-use-remap-path-prefix/104406)
- Don't do your final build on your local machine. Use GitHub CI or similar.

**Comment by mockersf** [comment]
By default Rust will include the build path in the binary

Even if you disable the logs, someone searching the executable could still find your name.
